### ISSUE TITLE
The railroad diagram blocked?

### ISSUE THREAD

**Issue by Qianyi951015** [apache/shardingsphere#30422]
Hi guys,
          I found that the railroad diagram in our DistSql document was blocked as the image shown below. Idk if it's my problem or any other problem. 

![image](https://github.com/apache/shardingsphere/assets/57847965/9fe560e4-5aac-4986-9572-ce19f9259754)

**Comment by RaigorJiang** [comment]
Yes, currently the railroad diagram cannot be displayed. I think it may be caused by the security policy of Apache.

**Comment by Qianyi951015** [comment]
> Yes, currently the railroad diagram cannot be displayed. I think it may be caused by the security policy of Apache.

Got it, Ty

**Comment by zzyReal666** [comment]
I think the issue can be closed.

**Comment by Qianyi951015** [comment]
> I think the issue can be closed.

But the railroad diagram still cannot be displayed. Do you have any good ideas?

**Comment by zzyReal666** [comment]
https://rr.red-dove.com/ui  This website may be able to help you, I generated the Railroad Diagram via grammar

**Comment by Qianyi951015** [comment]
> https://rr.red-dove.com/ui This website may be able to help you, I generated the Railroad Diagram via grammar.

It's not that I don't understand the syntax of railroad diagrams. The documents of these railroad diagrams were originally edited by me in 2022. What I expect is that users can see the railroad diagrams directly from our official documents, so that they can understand the DistSQL syntax more simply and clearly.
Do you have other ideas to display railroad diagrams in our official documents? Or can any other methods of explaining Distsql syntax help users easily understand instead of using railroad diagrams?

**Comment by zzyReal666** [comment]
> > https://rr.red-dove.com/ui这个网站也许可以帮助你，我通过语法生成了铁路图。
> 
> 不是我不懂铁路图的语法，这些铁路图的文档是我在2022年编辑的，我期望的是用户能直接从我们的官方文档中看到铁路图，这样他们就能更简单、更清楚地理解 DistSQL 的语法。 你还有其他想法在我们的官方文档中展示铁路图吗？或者除了使用铁路图之外，还有其他解释 Distsql 语法的方法可以帮助用户轻松理解吗？

Sorry, I misread your question. I thought you just wanted to see the Railroad Diagram.

**Comment by terrymanu** [comment]
Fixed
